python/pynessie/_endpoints.py

```diff
@@ -8,8 +8,10 @@
 
 import requests
 import simplejson as jsonlib
+from confuse import Configuration
```

Comment:
this was a set of functions to allow for composing functions w/o a class? Why make a class now? This class is now nearly identical to `NessieClient` so we either refactor into that class or keep these as functions

Reply:
I went back to individual function in `_endpoints` in cce999859b660e462b538b51b451f438e210ee6e (mainly to reduce the amount of changes in the PR overall).